tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
It's kinda lame. It's there to pass the compiler because before my changes there were no test on this repo.

Now as I look at it, I can probably exclude the `example` folder to begin with.

Will merge this and then fix that.